docs/developers/msgs/profiles/refuse-dtag-transfer-request.md

```diff
@@ -0,0 +1,37 @@
+# `MsgRefuseDTagTransferRequest`
+This message allows you to refuse a `DTag`'s transfer request made by a user.
```

Comment:
Actually that two names never convinced me a lot but since I didn't have a better idea in that moment I left them there knowing that sooner or later they would have been changed. So yeah, I think we can have those you suggest without any problem